internal/balancergroup/balancergroup.go

```diff
@@ -110,11 +110,7 @@ func (sbc *subBalancerWrapper) exitIdle() {
 		ei.ExitIdle()
 		return
 	}
-	for sc, b := range sbc.group.scToSubBalancer {
-		if b == sbc {
-			sc.Connect()
-		}
-	}
+	sbc.group.connect(sbc)
```

Comment:
So the balancer group calls into the wrapper calls back into the group?

Would it be better to put all this logic in the group's ExitIdle function, then?  Or return a bool and have ExitIdle do the `connect` work?

Reply:
Done.